src/index.html

```diff
@@ -15,8 +15,42 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+   <header class="header">
+      <img
+        class="logo"
+        src="images/logo.png"
+        alt="logo_moyo"
+      />
+      <nav class="nav-bar">
+        <a class="nav__link is-active">APPLE</a>
```

Comment:
```suggestion
      <nav class="nav-bar">
       <ul>
         <li>
          <a class="nav__link is-active">APPLE</a>
        </li>
```
try to rewrite this structure to be

Reply:
im done it but got wrong test. and im need help with this